src/libAtomVM/term.h

```diff
@@ -1078,11 +1078,11 @@ static inline term term_make_boxed_int64(avm_int64_t large_int64, Heap *heap)
         boxed_int[1] = large_int64;
     #elif BOXED_TERMS_REQUIRED_FOR_INT64 == 2
         #if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
-            boxed_int[1] = large_int64;
-            boxed_int[2] = large_int64 >> 32;
+            boxed_int[1] = (term) large_int64;
+            boxed_int[2] = (term) (large_int64 >> 32);
```

Comment:
Was it a warning?

Reply:
We can get rid of this change, I'm not sure it fixed anything.